python/pyspark/pandas/resample.py

```diff
@@ -130,19 +130,6 @@ def _resamplekey_type(self) -> DataType:
     def _agg_columns_scols(self) -> List[Column]:
         return [s.spark.column for s in self._agg_columns]
 
-    def get_make_interval(  # type: ignore[return]
```

Comment:
Just wondering if we skip some units intentionally here

Reply:
some units was ignored due to not invoked at that time, I am adding them back to make the helper function cover all pandas units :)

Reply:
Great thanks!